spec/controllers/chargeback_controller_spec.rb

```diff
@@ -110,4 +111,25 @@
       expect(flash_array.first[:message]).to include("rate is assigned and cannot be deleted")
     end
   end
+  context "#get_cis_all" do
+    elements_miq = %w(enterprise storage ext_management_system ems_cluster tenants)
+    elements_miq.each do |element|
+      it "returns names of instances of " + element do
+        names = {}
+        classtype =
+          if element == "enterprise"
+            MiqEnterprise
+          else
+            element.classify.constantize
+          end
+
+        classtype.all.each do |instance|
+          names[instance.id] = instance.name
+        end
+        controller.instance_variable_set(:@edit, :new => {:cbshow_typ => element}, :cb_assign => {})
```

Comment:
I just did the changes please check them in #7202 I'll merge that here after finish that PR. Thanks